### DISCUSSION TITLE
Export to CSV button on the form

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5704] [Feature Requests]
> It would be a nice feature to have a button on a form that would let a user download a pre-configured csv summary of the data displayed - we've been asked if there is a quick way to export data when a user is browsing through collection object forms after running a search.
> 
> I suspect the easiest implementation would be having a form object that connects to a saved query (in our case created via our admin account) that just shortcuts straight to the creation of the csv.
> 
> It would be nice to have the option to either download the record being viewed or download all the records from the "browse in form" link - this would be a bit redundant if you are viewing from the query builder but the "browse in form" link also appears in the Express Search as well.

Maybe this could be added to the meta menu? Seems like a really interesting idea!

**Requested By:** Dan Baker at CSIRO ([Asana Link](https://app.asana.com/0/1201972579963460/1203637920507462/f))

**Comment by melton-jason** [comment]
> Is there any way to make the "Generate Label” button on the Forms to generate a CSV file with the current collection object information instead of a PDF file?

**Requested By**:  Malca Boni and Eyal Ben-Hur at the NNHC at HUJI
